### ISSUE TITLE
tabbed: link multliple sets on page

### ISSUE THREAD

**Issue by casperdcl** [facelessuser/pymdown-extensions#1456]
Feature request:

```md
=== "A"
    The first letter.

=== "B"
    The second letter.

Some more text about both letters

=== "A"
    More about the first letter.

=== "B"
    More about the second letter.
```

Upon clicking a tab, *both* sets should update.

I've seen this on a website before but can't find the link now.

**Comment by facelessuser** [comment]
Yeah, this is something that can be implemented with additional JavaScript. There is no way to do this with pure CSS or with just HTML tags. We do not inject JavaScript in the final output though. So this wouldn't be built into the output. I could make a note in the docs of what an example JS solution would be though.

I know the MkDocs Material theme (a project that uses Pymdown Extensions) has a feature in their pipes (not sure if it is already out or not) to do just this. It uses/will use JavaScript to accomplish it as that is the only way.

If I get a chance, I'm more than happy to post a simple JavaScript example of how to accomplish this. Pull requests are also appreciated. Or even someone just wants to post a working example, I'm happy to place that in the docs.

**Comment by casperdcl** [comment]
ah I see it's a premium feature circa $3,000 away from being released to the general public :)

https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#linking-content-tabs

**Comment by facelessuser** [comment]
Yeah, it will make it to the public at some point. It's not hard to do with JavaScript though. For the most part, I leave that up to the user, but if I find some time, I can document a simple example, but I am open to people provide a simple snippet that I can include into the docs as well.

**Comment by facelessuser** [comment]
@casperdcl I have a rough solution:

![tabs](https://user-images.githubusercontent.com/1055125/134051563-040dfa31-cf19-4a73-866c-16177f62db28.gif)

You may have to format this to plain vanilla JS and such, but this is what I used. It doesn't save this across pages or anything like that. But I wanted to at least provide a very basic starting point. Hop that helps!

```js
const tabs = e => {
  if (e.target.matches('.tabbed-set > input[type=radio]')) {
    const label = e.target.nextSibling.innerHTML
    const labels = document.querySelectorAll('.tabbed-set > label')
    labels.forEach(el => {
      if (el.innerHTML === label) {
        el.previousSibling.checked = true;
      }
    })
  }
}

document.addEventListener("click", tabs)
```

**Comment by facelessuser** [comment]
We ended up adding this functionality into our own docs and documenting it. It will be officially in the docs on the next release.

The code we ended up using is found here: https://github.com/facelessuser/pymdown-extensions/blob/main/docs/src/js/tabbed.js#L41.

As we've shared a solution and documented this for future users, I will consider this issue resolved.

**Comment by casperdcl** [comment]
closed by #1417
